### PR TITLE
docs: Amexio description changed

### PR DESCRIPTION
**Pull request by rashmithakkar**
Amexio description changed

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by StephenFluin — APPROVED** [review]
Content looks good.

**Comment by atscott** [comment]
@rashmithakkar Could you accept the Google CLA so this can be merged?

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -414,7 +414,7 @@
             "url": "https://www.jqwidgets.com/angular/"
           },
           "amexio": {
-            "desc": "Amexio (Angular MetaMagic EXtensions for Inputs and Outputs)  is a rich set of Angular components powered by Bootstrap for Responsive Design. UI Components include Standard Form Components, Data Grids, Tree Grids, Tabs etc. Open Source (Apache 2 License) & Free and backed by MetaMagic Global Inc",
+            "desc": "Amexio is a rich set of Angular components powered by HTML5 & CSS3 for Responsive Web Design and 80+ built-in Material Design Themes. Amexio has 3 Editions, Standard, Enterprise and Creative. Std Edition consists of basic UI Components which include Grid, Tabs, Form Inputs etc, while Enterprise Edition consists of components like Calendar, Tree Tabs, Social Media Logins (Facebook, GitHub, Twitter etc) and Creative Edtion is focused building elegant and beautiful websites. With more than 200+ components/features and all the editions are open-sourced and free based on Apache 2 License.",
```
Comment (jbogarthyde) [inline aio/content/marketing/resources.json]: Some changes to conform with doc authoring guidelines:
- Please keep the expansion of "Amexio". 
- Amexio has 3 editions: Standard, Enterprise and Creative. 
- Std -> Standard edition (don't abbreviate, don't capitalize "edition")
- etc -> and so on
- use separate sentences for the descriptions of the 3 editions
- typo: Creative Edtion -> Creative edition
"websites with more than 200+ components/features. All editions are open-sourced and free, based on..."

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
This still needs the CLA, but in the meantime, I'm suggesting some editorial changes to conform to Google and Angular documentation standards.

**Comment by AndrewKushnir** [comment]
Closing this PR due to the lack of response. Please reopen if you get a chance to look into it. Thank you.

**Pull request closed without merging**